style.css

```diff
@@ -117,3 +118,289 @@ html {
   display: flex;
   padding-left: 0;
 }
+
+.works {
+  background-color: #f7f7f9;
+  margin: 0;
+  display: grid;
+  grid-template-rows: auto;
+  padding-top: 80px;
+}
+
+.works-flex {
+  height: auto;
+  background: rgba(255, 255, 255, 1);
+  margin: 0 15px 80px 15px;
+  border-style: solid;
+  border-color: rgb(219, 214, 214);
+  border-width: 1px;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.works-flex-item {
+  margin: 0 15px 0 15px;
+  justify-content: center;
+}
+
+.work-image {
+  align-self: center;
+  border-radius: 10px;
+  margin-top: 15px;
+}
+
+.intro-works-text {
+  font-weight: 650;
+  padding-top: 10px;
+  font-size: 35px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  margin: 0 0 0 0;
+}
+
+.titles-container {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0;
+  justify-self: flex-start;
+  padding-top: 10px;
+  width: 100%;
+}
+
+.list-main {
+  list-style-type: none;
+  font-weight: 600;
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  color: #344563;
+  padding-right: 25px;
+  flex: 1;
+  margin-left: 30px;
+}
+
+.titles-item {
+  font-weight: 500;
+  font-size: 12px;
+  font-family: 'Poppins', sans-serif;
+  color: rgba(122, 134, 154, 1);
+  list-style-image: url("Counterping_1.png");
+  flex: 2;
+}
+
+.works-description {
+  font-weight: normal;
+  font-size: 15px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  padding-top: 10px;
+  margin-top: 0;
+}
+
+.lang-container {
+  padding-top: 25px;
+  padding-left: 0;
+  width: 70%;
+  margin: 0;
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  list-style: none;
+}
+
+.lang {
+  font-size: 15px;
+  border-radius: 8px;
+  padding: 4px 12px 4px;
+  align-items: center;
+  background: #ebebff;
+  color: #6070ff;
+}
+
+.btn-see-project {
+  text-decoration: none;
+  color: #4053fc;
+  border: 1px solid #396df2;
+  border-radius: 4px;
+  padding: 7px;
+  line-height: 30px;
+  font-size: 17px;
+}
+
+.btn-see-project:disabled {
+  font-weight: 300;
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
+  background: #ebebff;
+  color: #808080;
+  border: 2px solid #808080;
+  border-radius: 4px;
+  padding: 7px;
+}
+
+a.btn-see-project:hover {
+  border: 1px solid #396df2;
+  padding: 7px;
+  border-radius: 4px;
+  font-size: 17px;
+  font-weight: 300;
+  background: #396df2;
+  color: #ebebff;
+  font-family: 'Poppins', sans-serif;
+}
+
+a.btn-see-project:active {
+  padding: 7px;
+  border-radius: 4px;
+  font-size: 17px;
+  font-weight: 300;
+  background: #000080;
+  color: #ebebff;
+  font-family: 'Poppins', sans-serif;
+}
+
+.btn-see {
+  padding-top: 30px;
+  height: 90px;
+}
+
+.intro-text-about {
+  font-weight: 700;
+  font-size: 40px;
+  font-family: 'Poppins', sans-serif;
+  color: #091e42;
+  margin-bottom: 5px;
+}
+
+a.btn-see-resume {
+  text-decoration: none;
+}
```

Comment:
- [ ] [OPTIONAL] I strongly suggest adding `border-radius` to this button in order to match the Figma template. I will mark this as [OPTIONAL] since this has not been addressed by the previous reviewer, but it could be a good idea to update this :smiley: :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/143593309-d64f3ceb-e5c7-45c6-8925-2399736f0ec5.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/143593361-2e8a7799-6b03-456e-83fd-0d75fd266e9e.png)